src/site/topics/bags/bag-implementations.rst

```diff
@@ -327,9 +327,9 @@ ArraySortedBag
     * The method ``compareTo`` is implemented for the type
 
 * If we call ``x.compareTo(y)``
-    * Return a negative integer if ``y < x``
-    * Return zero if ``y == x``
-    * Return a positive integer if ``y > x``
+    * Return a negative integer if ``x < y``
+    * Return zero if ``x == y``
+    * Return a positive integer if ``x > y``
```

Comment:
It do always be a little confusing. Think of `compareTo(..) < 0` like the `<` operator itself.